README.md

```
@@ -39,11 +39,12 @@ version = '0.0.2-SNAPSHOT'
 ## 📝 버전
 
-| 버전 | 변경 내용 |
-|------|-----------|
+| 버전               | 변경 내용                                                                                                                                                                                                                                      |
+|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `0.0.1-SNAPSHOT` | • 공통 응답 구조 추가 (`ApiResponse`, `ErrorCode`, `SuccessCode`)<br>• 공통 예외 처리 추가 (`BusinessException`, `GlobalExceptionHandler`)<br>• JPA 기본 엔티티 및 설정 추가 (`BaseEntity`, `BaseUserEntity`, `JpaConfig`)<br>• 자동 설정 등록 (`CommonAutoConfiguration`) |
-| `0.0.2-SNAPSHOT` | • Feign 설정 추가 (`FeignConfig`, `FeignErrorDecoder`)<br>• PageableResolver 추가 (`CustomPageableResolver`, `WebConfig`)<br>• `JsonUtil` 추가 (JSON 직렬화/역직렬화 정적 유틸)<br>• `ObjectMapper` 빈 등록 (`JavaTimeModule`, `FAIL_ON_UNKNOWN_PROPERTIES` 설정) |
-| `0.0.3-SNAPSHOT` | • JPA 관련 코드 제거 (`BaseEntity`, `BaseUserEntity`, `JpaConfig`) → `common-jpa`로 분리 |
+| `0.0.2-SNAPSHOT` | • Feign 설정 추가 (`FeignConfig`, `FeignErrorDecoder`)<br>• PageableResolver 추가 (`CustomPageableResolver`, `WebConfig`)<br>• `JsonUtil` 추가 (JSON 직렬화/역직렬화 정적 유틸)<br>• `ObjectMapper` 빈 등록 (`JavaTimeModule`, `FAIL_ON_UNKNOWN_PROPERTIES` 설정)  |
+| `0.0.3-SNAPSHOT` | • JPA 관련 코드 제거 (`BaseEntity`, `BaseUserEntity`, `JpaConfig`) → `common-jpa`로 분리                                                                                                                                                            |
+| `0.0.4-SNAPSHOT` | • `UserContext` 추가 (Gateway 주입 헤더 -> 인증 사용자 정보 DTO)<br> • `UserContextArgumentResolver` 추가 (컨트롤러 파라미터 자동 주입) |                                                                                                                              |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)